src/integration/common/storyPromo.js

```diff
@@ -0,0 +1,39 @@
+export default () => {
+  describe('Story Promo', () => {
+    const section = document.querySelector('section');
```

Comment:
We might be able to use a more specific selector to get the story promo - perhaps best not to assume it's in a section (because that will only work for FrontPage, IndexPage and FeatureIndexPage